deploy/helm/sumologic/values.yaml

```diff
@@ -3366,16 +3370,39 @@ otelagent:
       extensions: [health_check, memory_ballast]
       pipelines:
         traces:
-          receivers: [jaeger, opencensus, otlp, zipkin]
+          receivers: [jaeger, opencensus, otlp, otlp/deprecated, zipkin]
           processors: [memory_limiter, k8s_tagger, batch]
           exporters: [otlp]
         metrics:
-          receivers: [otlp]
+          receivers: [otlp, otlp/deprecated]
           processors: [memory_limiter, k8s_tagger, batch]
           exporters: [otlp]
 
 ## Configure otelcol
 otelcol:
+  ## Set the _sourceName metadata field in Sumo Logic.
+  source_metadata:
```

Comment:
Is there a reason why the `source_metadata` property uses snake_case while the other properties use camelCase?

Reply:
nah, there's no :) will change it 👍